integration-tests/sql/src/test/java/org/apache/camel/quarkus/component/sql/it/SqlTest.java

```diff
@@ -67,6 +71,7 @@ public void testSqlComponent() {
     }
 
     @Test
+    @DisabledIfSystemProperty(named = "SQL_JDBC_DB_KIND", matches = "[^h][^2].*")
```

Comment:
If you think this can be made to work for other DBs, please file an issue and paste the link here.

Reply:
Makes sense, I'll fix it.

Reply:
done